examples/Makefile.am

```diff
@@ -6,11 +6,11 @@
 #
 
 examplesdir = $(pkgdatadir)/examples
-dist_examples_DATA = \
+dist_examples_DATA =  \
```

Comment:
maybe rename "ucx_hello_world.{h,c}" to "hello_world_common.{h,c}" ?

Reply:
there is no ucx_hello_world.**c** file, and there are hello world sources:
```ucp_hello_world.c  uct_hello_world.c  ucx_hello_world.h```
renaming header file to ```hello_world_common.h``` will look strange:
```ucp_hello_world.c  uct_hello_world.c hello_world_common.h```

what do you think?

Reply:
i suggest something like:
ucp_hello_world.c uct_hello_world.c hello_world_common.h
or
ucp_hello_world.c uct_hello_world.c common.h
or
ucp_hello_world.c uct_hello_world.c hello_world_util.h

Reply:
ok, done